components/closebutton/stories/closebutton.stories.js

```diff
@@ -1,10 +1,13 @@
+import { Sizes } from "@spectrum-css/preview/decorators";
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
 import { isDisabled, size, staticColor } from "@spectrum-css/preview/types";
 import pkgJson from "../package.json";
 import { CloseButtonGroup } from "./closebutton.test.js";
+import { CloseButtonExample, Template } from "./template.js";
 
 /**
- * A button used to close or dismiss components.
+ * Close button is used to close or dismiss its parent component. It is used inside of other components such
+ * as [toast](?path=/docs/components-toast--docs) and [action bar](?path=/docs/components-action-bar--docs).
```

Comment:
```suggestion
 * as [Toast](?path=/docs/components-toast--docs) and [Action bar](?path=/docs/components-action-bar--docs).
```

Reply:
I want to capitalize these too, but the initial guidance I got from the content writing team was to not capitalize them within the sentence (which is also how the guidelines appear to do it).

Reply:
Oh _interesting_ - thanks for the clarification.